src/components/Movie/style.css

```diff
@@ -0,0 +1,38 @@
+h2,
+h3 {
+    color: tomato;
+    margin: 15px 0px;
+}
+
+h2 {
+    font-size: 1.5em;
+    margin-top: 0px;
+}
+
+h3 {font-size: 1em;}
+
+p {margin: 5px;}
```

Comment:
zase bacha na to, tímhle styluješ všechny h2, h3, p v celé aplikaci (to většinou nechceš). Pokud to má být jen pro tuhle komponentu, psal bych něco jako
```
.movie__detail p {margin: 5px;}
```